Library/include/CSP/Systems/SystemsManager.h

```diff
@@ -131,6 +132,10 @@ class CSP_API CSP_NO_DISPOSE SystemsManager
 	/// @return ECommerceSystem : pointer to the ECommerce system class
 	ECommerceSystem* GetECommerceSystem();
 
+	/// @brief Retrieves the Quota system.
+	/// @return QuotaSystem : pointer to the Quota system class
+	QuotaSystem* GetQuotaSystem();
```

Comment:
Are there C# tests needed for this ? As there are other systems which also include C# tests as part of the introduction of the system.